### PR TITLE
files-review-p1

### PR DESCRIPTION
**Pull request by coolboy**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @coolboy has signed the CLA. Thanks for the pull request!

File: Box.V2/Managers/BoxFilesManager.cs [hunk 1]
```diff
@@ -43,12 +43,12 @@ public BoxFilesManager(IBoxConfig config, IBoxService service, IBoxConverter con
...
-        /// Returns the stream of the requested file
+        /// Returns the stream of the requested file.
         /// </summary>
-        /// <param name="id">Id of the file to download</param>
+        /// <param name="id">Id of the file to download.</param>
         /// <param name="versionId">The ID specific version of this file to download.</param>
-        /// <param name="timeout">Optional timeout for response</param>
-        /// <returns>MemoryStream of the requested file</returns>
+        /// <param name="timeout">Optional timeout for response.</param>
+        /// <returns>MemoryStream of the requested file.</returns>
```
Comment (cburnette) [inline Box.V2/Managers/BoxFilesManager.cs]: change to Stream

File: Box.V2/Managers/BoxFilesManager.cs [hunk 2]
```diff
@@ -353,9 +351,11 @@ private string HexStringFromBytes(byte[] bytes)
         /// <summary>
         /// Used to create a shared link for this particular file. Please see here for more information on the permissions available for shared links. 
         /// </summary>
-        /// <param name="id"></param>
-        /// <param name="sharedLinkRequest"></param>
-        /// <returns></returns>
+        /// <param name="id">Id of the file</param>
```
Comment (cburnette) [inline Box.V2/Managers/BoxFilesManager.cs]: add periods

File: Box.V2/Managers/BoxFilesManager.cs [hunk 3]
```diff
@@ -414,15 +414,15 @@ private string HexStringFromBytes(byte[] bytes)
...
-        /// <param name="minHeight">The minimum height of the thumbnail</param>
-        /// <param name="minWidth">The minimum width of the thumbnail</param>
-        /// <param name="maxHeight">The maximum height of the thumbnail</param>
-        /// <param name="maxWidth">The maximum width of the thumbnail</param>
+        /// <param name="id">Id of the file.</param>
+        /// <param name="minHeight">The minimum height of the thumbnail.</param>
+        /// <param name="minWidth">The minimum width of the thumbnail.</param>
+        /// <param name="maxHeight">The maximum height of the thumbnail.</param>
+        /// <param name="maxWidth">The maximum width of the thumbnail.</param>
         /// <param name="handleRetry">specifies whether the method handles retries. If true, then the method would retry the call if the HTTP response is 'Accepted'. The delay for the retry is determined
```
Comment (cburnette) [inline Box.V2/Managers/BoxFilesManager.cs]: capitalize Specifies

File: Box.V2/Managers/BoxFilesManager.cs [hunk 4]
```diff
@@ -414,15 +414,15 @@ private string HexStringFromBytes(byte[] bytes)
...
+        /// <param name="minWidth">The minimum width of the thumbnail.</param>
+        /// <param name="maxHeight">The maximum height of the thumbnail.</param>
+        /// <param name="maxWidth">The maximum width of the thumbnail.</param>
         /// <param name="handleRetry">specifies whether the method handles retries. If true, then the method would retry the call if the HTTP response is 'Accepted'. The delay for the retry is determined 
-        /// by the RetryAfter header, or if that header is not set, by the constant DefaultRetryDelay</param>
-        /// <param name="throttle">Whether the requests will be throttled. Recommended to be left true to prevent spamming the server</param>
-        /// <returns>Contents of thumbnail</returns>
+        /// by the RetryAfter header, or if that header is not set, by the constant DefaultRetryDelay.</param>
+        /// <param name="throttle">Whether the requests will be throttled. Recommended to be left true to prevent spamming the server.</param>
+        /// <returns>Contents of thumbnail.</returns>
```
Comment (cburnette) [inline Box.V2/Managers/BoxFilesManager.cs]: as Stream

File: Box.V2/Managers/BoxFilesManager.cs (hunk 1 above)
Comment (coolboy) [inline Box.V2/Managers/BoxFilesManager.cs]: done

File: Box.V2/Managers/BoxFilesManager.cs (hunk 3 above)
Comment (coolboy) [inline Box.V2/Managers/BoxFilesManager.cs]: done

File: Box.V2/Managers/BoxFilesManager.cs (hunk 4 above)
Comment (coolboy) [inline Box.V2/Managers/BoxFilesManager.cs]: done

**Pull request merged**